core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/InstalledIntelliJ.scala

```diff
@@ -42,7 +42,8 @@ sealed abstract class InstalledIntelliJ(root: Path, probePaths: IdeProbePaths, c
       "-Djb.consents.confirmation.enabled=false"
     )
 
-    val vmOptions = implementationSpecificVmOptions ++ baseVMOptions ++ DebugMode.vmOption ++ config.vmOptions
+    val vmOptions =
+      implementationSpecificVmOptions ++ baseVMOptions ++ DebugMode.vmOption(config.debug) ++ config.vmOptions
```

Comment:
```suggestion
      implementationSpecificVmOptions ++ baseVMOptions ++ DebugMode.vmOptions(config.debug) ++ config.vmOptions
```

As it's an implementation details that currently it can only return 0 or 1 options; could as well return 2+